Sources/Exporters/OpenTelemetryProtocol/trace/OtlpHTttpTraceExporter.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import OpenTelemetrySdk
+import Foundation
+
+public func defaultOltpHttpTracesEndpoint() -> URL {
+    URL(string: "http://localhost:4318/v1/traces")!
+}
+
+public class OtlpHttpTraceExporter: SpanExporter {
+    let endpoint: URL
+    private let httpClient: HTTPClient
+    var pendingSpans: [SpanData] = []
+     
+    public init(endpoint: URL = defaultOltpHttpTracesEndpoint()) {
+        self.endpoint = endpoint
+        self.httpClient = HTTPClient()
+    }
```

Comment:
```suggestion
       public init(endpoint: URL = defaultOltpHttpTracesEndpoint(), useSession: URLSession? = nil) {
        self.endpoint = endpoint
        if let providedSession = useSession {
            self.httpClient = HTTPClient(session: providedSession)
        } else {
            self.httpClient = HTTPClient()
        }
    }

Reply:
Something like this allows reusing existing URLSession from the user. Would need changing all exporters